validator/client/src/main/java/tech/pegasys/teku/validator/client/loader/ValidatorLoader.java

```diff
@@ -92,11 +100,44 @@ static ValidatorLoader create(
           metricsSystem,
           validatorSources);
       addLocalValidatorSource(spec, config, slashingProtector, asyncRunner, validatorSources);
+      if (validatorPath.isPresent()) {
+        addMutableValidatorSource(
+            spec, config, slashingProtector, asyncRunner, validatorSources, validatorPath);
+      }
```

Comment:
this suggests we can just pass the .get() of the validator path and not pass down an optional, which might simplify my below comment.